PFR/ForMathlib/Entropy/RuzsaSetDist.lean

```diff
@@ -90,7 +90,7 @@ lemma isUniform_iff_uniform_dist {Ω : Type*} [mΩ : MeasurableSpace Ω] {μ: Me
           simp only [Set.Finite.mem_toFinset, Set.mem_inter_iff, AHf] at hx
           simp [hx]
         intro x hx
-        simp at hx
+        simp only [Set.Finite.mem_toFinset, Set.mem_inter_iff, not_and] at hx
         simpa
```

Comment:
This simp is preceding a simpa, which is flexible

Reply:
Done. Thanks.